ckanext/datajson/tests/datajson-samples/geospatial.data.json

```diff
@@ -156,7 +96,7 @@
           "@type": "dcat:Distribution",
           "title": "[Anything valid here] Original Metadata",
           "downloadURL": "https://data.usgs.gov/metadata/Geologic_Hazards_Science_Center/5ec70f9d82ce476925eddc9e.xml",
-          "conformsTo": "https://www.fgdc.gov/schemas/metadata/fgdc-std-001-1998.xsd",
+          "conformsTo": "http://www.fgdc.gov/schemas/metadata",
```

Comment:
Why convert this to an `http` link?

Reply:
It redirected to `https` anyway...

![image](https://user-images.githubusercontent.com/85196563/216403665-41575eae-30d0-42f9-b31b-a6c397d1feb8.png)